db/migrate/20160322195653_move_switch_host_to_jointable.rb

```diff
@@ -0,0 +1,25 @@
+class MoveSwitchHostToJointable < ActiveRecord::Migration[5.0]
```

Comment:
Every time I see this 5.0 versioned migration number, it makes me happy.  Thanks @matthewd :+1: